src/classes/FACI_CannotDelete_TEST.cls

```diff
@@ -0,0 +1,455 @@
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group Facility
+* @group-content ../../ApexDocContent/Facility.htm
+* @description Tests for FACI_CannotDelete_TDTM
+*/
+@isTest
+private class FACI_CannotDelete_TEST {
+    /*********************************************************************************************************
+    * @description Retrieves the Administrative record type Id. 
+    */
+    public static String adminAccRecTypeId = UTIL_Describe.getAdminAccRecTypeID(); 
+    
+    /*********************************************************************************************************
+    * @description Retrieves the Academic Program record type Id. 
+    */
+    public static String academicAccRecTypeId = UTIL_Describe.getAcademicAccRecTypeID(); 
+    
+    /*********************************************************************************************************
+    * @description Test method to test if Prevent_Facility_Deletion__c is enabled in Hierarchy Settings, and
+    * Facilities has a Facilities record associated to it, then it cannot be deleted.
+    */
+    @isTest
+    public static void cannotDeleteFacilitiesWithFacilities() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                        (Account_Processor__c = FACI_CannotDelete_TEST.adminAccRecTypeId,
+                                                        Prevent_Facility_Deletion__c = True));
+
+        List<Facility__c> parentFacilitiesToInsert = new List<Facility__c>(); 
+        Facility__c parentFacility = UTIL_UnitTestData_TEST.getFacility(); 
+        Facility__c parentFacility2 = UTIL_UnitTestData_TEST.getFacility();
+        parentFacilitiesToInsert.add(parentFacility); 
+        parentFacilitiesToInsert.add(parentFacility2); 
+        insert parentFacilitiesToInsert; 
+
+        Facility__c childFacility1 = UTIL_UnitTestData_TEST.getFacility(); 
+        Facility__c childFacility2 = UTIL_UnitTestData_TEST.getFacility(); 
+        childFacility1.Parent_Facility__c = parentFacility.Id; 
+        childFacility2.Parent_Facility__c = parentFacility2.Id; 
+        insert childFacility1;
```

Comment:
@kdy618 In the future consider collapsing these into a list insert for a single DML statement that will increase the speed of our unit tests.